src/components/LayoutContainer.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from "react";
+import {
+  Icon,
+  Image,
+  Menu,
+  Responsive,
+  Segment,
+  Sidebar,
+  Container
+} from "semantic-ui-react";
+
+import { Page } from "../constants";
+import logo from "../images/reel-politik-logo-1.png";
+import { string } from "prop-types";
+
+interface propTypes {
+  activePage: Page;
+  className?: string;
+  style?: React.CSSProperties;
+}
+
+const Logo = () => (
+  <Image src={logo} style={{ height: "3em", margin: "0.5em" }} />
+);
+
+const LayoutContainer: React.SFC<propTypes> = ({
+  activePage,
+  children,
+  className,
+  style
+}): JSX.Element => {
+  const [sidebarOpen, setSidebarOpen] = useState(false);
+
+  function handleSidebarHide() {
+    setSidebarOpen(false);
+  }
+
+  function handleToggle() {
+    setSidebarOpen(!sidebarOpen);
+  }
+  return (
+    <Sidebar.Pushable {...className} {...style}>
+      <Sidebar
+        animation="overlay"
+        as={Menu}
+        inverted
+        onHide={handleSidebarHide}
+        vertical
```

Comment:
Huh, weird that you have `vertical` here cause it's showing up as horizontal on my screen

![image](https://user-images.githubusercontent.com/11564650/57575918-06419600-7409-11e9-91a9-52a193b0fe30.png)

Reply:
AD, vertical actually applies to the sidebar, refactored.